### DISCUSSION TITLE
Thank you for this

### DISCUSSION THREAD

**Discussion by em-777** [bakkeby/dwm-commented#4] [General]
I've always wanted to start using and customizing dwm, but the lack of documentation made the task too daunting. Your detailed comments are incredibly helpful.

**Comment by bakkeby** [comment]
You are very welcome. I am just happy that it is helpful to some.

**Reply by muh-5** [reply]
It's very helpful, and it makes patching more understandable.

Thank you, @bakkeby , you did great work.
